### DISCUSSION TITLE
Hot fix alpha release

### DISCUSSION THREAD

**Discussion by rrjanbiah** [vazco/uniforms#1238] [Ideas]
Is it possible to have some alpha release especially with the fixes such as https://github.com/vazco/uniforms/issues/1223 ? Thank you

**Comment by radekmie** [comment]
Hi @rrjanbiah. There is one planned for later today.

**Reply by rrjanbiah** [reply]
Awesome. Thank you

**Reply by rrjanbiah** [reply]
@radekmie Is that getting released in a different channel? Or am I missing anything? Thank you

**Reply by radekmie** [reply]
No, it's released as usual and [present on npm](https://www.npmjs.com/package/uniforms). Is there a problem?

**Reply by rrjanbiah** [reply]
Ah, thank you. My bad, I was watching the milestone. Thanks again
